presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -646,14 +646,19 @@ public Plan createPlan(Session session, @Language("SQL") String sql)
 
     public Plan createPlan(Session session, @Language("SQL") String sql, LogicalPlanner.Stage stage)
     {
+        return createPlan(session, sql, stage, false);
+    }
+
+    public Plan createPlan(Session session, @Language("SQL") String sql, LogicalPlanner.Stage stage, boolean distributed)
+    {
         Statement statement = unwrapExecuteStatement(sqlParser.createStatement(sql), sqlParser, session);
 
         assertFormattedSql(sqlParser, statement);
 
         FeaturesConfig featuresConfig = new FeaturesConfig()
                 .setDistributedIndexJoinsEnabled(false)
                 .setOptimizeHashGeneration(true);
-        PlanOptimizers planOptimizers = new PlanOptimizers(metadata, sqlParser, featuresConfig, true);
```

Comment:
`LocalQueryRunner` can also create plan for a given query. I added option for the plan generation to create plan with Exchanges to test distributed plans. Such distributed plans are not executed by `LocalQueryRunner`